packages/web/src/features/languages/InviteLanguageMemberView.tsx

```diff
@@ -63,12 +67,13 @@ export default function InviteLanguageMemberView() {
               {t('users:email').toUpperCase()}
             </FormLabel>
             <TextInput
+              {...formContext.register('email', {
+                required: true,
+              })}
               id="email"
```

Comment:
For HTML inputs and simple components that wrap HTML inputs, we can just use register